cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -33,6 +39,9 @@ pub struct CincinnatiGraphFetchPlugin {
     /// The upstream from which to fetch the graph
     pub upstream: String,
 
+    /// graph-builder connection timeout in seconds.
+    pub gb_timeout: u64,
```

Comment:
We try to use rich types everywhere, so this should be a `Duration`

Reply:
hmm, in fact we don't need it here - timeout is set once during `Client` build phase in `try_new`

Reply:
Even better ;-)